hw/vendor/patches/pulp_riscv_dbg/0001-Use-lowrisc-instead-of-PULP-primitives.patch

```diff
@@ -37,20 +37,20 @@ index 350c6c5..38d7b51 100644
 -  logic        resp_queue_empty;
```

Comment:
Did you directly edit this file instead of regenerating the patch? In the interest of time, I think this is okay.

Reply:
I think it would still be good to use the patching repository for this so that this can easily be generated automatically. It looks like this is already set up in the vendor file:
https://github.com/lowRISC/opentitan/blob/bcab2e16acc40f1c6e0445e26b90f720717ea81d/hw/vendor/pulp_riscv_dbg.vendor.hjson#L14-L18
So it would just be a matter of moving the patch to the ot branch [here](https://github.com/lowRISC/riscv-dbg/tree/ot) and invoking the vendor tool with the `--refresh-patches` switch:
```
$ util/vendor.py --refresh-patches --update --commit
```

Reply:
I tried that but it failed. I think [our patch repository](https://github.com/lowRISC/riscv-dbg/tree/ot) does not include patches from at least the last two PRs updating `hw/vendor/pulp_riscv_dbg`: https://github.com/lowRISC/opentitan/pull/12613, https://github.com/lowRISC/opentitan/pull/12728. In the interest of time, I did not spend the effort to first update the patch repository and port the patches there. But if you think that should be done now, I'll do it.

Reply:
I understand. Your approach is reasonable. Let's move forward and leave the cleanup for another time.

Reply:
I created https://github.com/lowRISC/opentitan/issues/15506 to track this

Reply:
Ok, SGTM!